### PR TITLE
Fix linchpin distiller to render values of every server provisioned

### PR DESCRIPTION
**Pull request by samvarankashyap**
Fixes issue #573

### CODE REVIEW

File: linchpin/cli/__init__.py [hunk 1]
```diff
@@ -305,34 +301,44 @@ def lp_up(self, targets=(), run_id=None, tx_id=None):
...
         return_code, return_data = self._execute_action('up',
                                                         targets,
                                                         run_id=run_id,
                                                         tx_id=tx_id)
 
+        # Distill data
         new_tx_id = return_data.keys()[0]
 
-        # This is what the API allows.
+        # Thsi is what the API allows.
```
Comment (p3ck) [inline linchpin/cli/__init__.py]: nit - spelling

File: linchpin/cli/__init__.py [hunk 2]
```diff
@@ -183,7 +183,7 @@ def _write_distilled_context(self, run_data):
         This method takes all of the provided run_data, loops through the
         distiller section of the linchpin.constants and writes out the
         linchpin.context (name TBD) to the 'resources' directory.
-
+
```
Comment (herlo) [inline linchpin/cli/__init__.py]: flake8 failed on these. The extra line is fine, but whitespace on a line by itself is not.

**Comment by samvarankashyap** [comment]
[test]

**Comment by samvarankashyap** [comment]
[merge]

**Pull request merged**